_layouts/production.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+---
+  <div class="production-posts">
+      <div class="production-post spacing">
+        <h1>{{ page.title }}</h1>
+          <b><span class="date">
+            {{ page.dates }} | {{page.location}}
+          </span></b>
+        </p>
+        {{ page.synopsis }}
+        <h3>PRODUCTION TEAM:</h3>
+        <h4>Director: <a href="{{page.director_bio_url}}">{{page.director}}</a>
+        <h4>Cast:</h4>
+
+        {% for cast in page.cast %}
+          {% if cast.actor_bio_url %}
+            <a href="{{cast.actor_bio_url}}">{{cast.actor}}</a> as {{cast.role}}
```

Comment:
If these are in a list or a paragraph tag, you'd eliminate the `<br>`. I think this is a better way, semantically, to convey it's a list.